automation_tools/__init__.py

```diff
@@ -1763,8 +1763,8 @@ def upstream_install(admin_password=None, run_katello_installer=True):
             '-e foreman_installer_skip_installer=True '
             'playbooks/katello.yml'.format(
                 '-e puppet_repositories_version=4' if puppet4 else '',
-                '-e foreman_repositories_use_koji=True' if use_koji else '',
-                '-e katello_repositories_use_koji=True' if use_koji else '',
+                '-e foreman_repositories_environment=staging' if koji else '',
+                '-e katello_repositories_environment=staging' if koji else '',
```

Comment:
I did not get a chance to test this out, so is it that by default, it is `nightly/release` if `foreman_repositories_environment=` is not specified?

Reply:
implicit default is 'release' so no need to be explicit about it

Reply:
ok, just wanted to confirm.